supporting_content/dimensions/region.md

```diff
@@ -0,0 +1,69 @@
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider | Data set                 | Column            |
+|----------|--------------------------|-------------------|
+| AWS      | CUR                      | Product_region    |
+| GCP      | Big Query Billing Export | Location.location |
```

Comment:
Needs to be changed across the board from GCP to Google Cloud - might be worth someone doing a global find and replace to ensure consistency
```suggestion
| GCP      | Big Query Billing Export | Location.location |
```